src/server/enrollment-system.js

```diff
@@ -1094,13 +1092,11 @@ function veteranToAssociationCollection(veteran) {
   const spouse = spouseToAssociation(veteran);
   associations.concat(children);
   if (spouse) { associations.push(spouse); }
-
   if (_.isEmpty(associations)) {
     return undefined;
   }
-  const result = associations.map((association) => { return { association }; });
-  console.log(result);
-  return result;
+  const result = associations.length === 1 ? associations.pop() : associations;
+  return { association: result };
```

Comment:
I think the SOAP library will handle this correctly if it's left as a single item array, but I'll write some tests around it to verify